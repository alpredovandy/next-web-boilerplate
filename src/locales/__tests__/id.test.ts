import commonId from "../id";

describe("Test locales: id", () => {
  it("Snapshot", () => {
    expect(commonId).toMatchSnapshot();
  });
});
