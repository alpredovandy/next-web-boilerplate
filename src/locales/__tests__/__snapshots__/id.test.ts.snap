// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test locales: id Snapshot 1`] = `
{
  "common": {
    "validation": {
      "required": "{name} wajib diisi.",
    },
  },
  "components": {
    "calendar": {
      "acceptText": "Terapkan",
      "cancelText": "Batalkan",
    },
    "timePicker": {
      "cancelText": "Batal",
      "labels": [
        "Jam",
        "Menit",
      ],
      "toolbarTitle": "Pilih waktu",
    },
  },
  "fiveHundred": {
    "action": "WhatsApp",
    "description": "Sistem Rey sedang mengalami masalah. Mohon coba kembli dalam beberapa saat atau hubungi kami via",
    "title": "Maaf, Sistem Sedang Error",
  },
  "fourOFour": {
    "action": "Beranda",
    "description": "Halaman ini tidak ada di Rey. Yuk, periksa ulang link tujuan kamu atau kembali ke",
    "title": "Yah, Halaman Tidak Ditemukan",
  },
}
`;
